src/DurableTask.Netherite/TransportLayer/EventHubs/EventHubsProcessor.cs

```diff
@@ -226,8 +244,9 @@ async Task<PartitionIncarnation> StartPartitionAsync(PartitionIncarnation prior
                 // the partition startup was canceled
                 this.traceHelper.LogDebug("EventHubsProcessor {eventHubName}/{eventHubPartition} canceled partition startup (incarnation {incarnation})", this.eventHubName, this.eventHubPartition, c.Incarnation);
             }
-            catch (Exception e) when (!Utils.IsFatal(e))
+            catch (Exception e)
```

Comment:
Why did we remove the `IsFatal` check?

Reply:
This snuck in when I was separating the PRs during my local development. It should actually be part of a different PR. Will remove from this one.